test/gui/shared/steps/steps.py

```diff
@@ -20,6 +20,29 @@
 confFilePath = confdir + 'owncloud.cfg'
 socketConnect = None
 
+passwords = {
+    'alt1': '1234',
+    'alt2': 'AaBb2Cc3Dd4',
+    'alt3': 'aVeryLongPassword42TheMeaningOfLife'
+}
+
+defaultUsers = {
+    'Alice': {
+        'displayname': 'Alice Hansen',
+        'password': passwords['alt1'],
+        'email': 'alice@example.org'
+    },
+    'Brian': {
+        'displayname': 'Brian Murphy',
+        'password': passwords['alt2'],
+        'email': 'brian@example.org'
+    },
+    'Carol': {
+        'displayname': 'Carol King',
+        'password': passwords['alt3'],
+        'email': 'carol@example.org'
+    }
+}
```

Comment:
note: in future that information should come out of the middleware see https://github.com/owncloud/owncloud-test-middleware/issues/16